PED/SetPedHeadBlendData.md

```diff
@@ -8,11 +8,11 @@ ns: PED
 void SET_PED_HEAD_BLEND_DATA(Ped ped, int shapeFirstID, int shapeSecondID, int shapeThirdID, int skinFirstID, int skinSecondID, int skinThirdID, float shapeMix, float skinMix, float thirdMix, BOOL isParent);
 ```
 
-For more info please refer to [this](https://gtaforums.com/topic/858970-all-gtao-face-ids-pedset-ped-head-blend-data-explained) topic.
+For more info and the list of faceIDs please refer to [this](https://gtaforums.com/topic/858970-all-gtao-face-ids-pedset-ped-head-blend-data-explained) topic. Note that the Skin and Shape IDs are shared. This native will use this same list for both Skin and Shape IDs.
```

Comment:
As @PsychoShock said, it would be better to have these things (such as images and others) in the game references section, rather than linking off-site. What if these resources go down?

Reply:
Do you prefer adding a new page on [docs](https://github.com/citizenfx/fivem-docs/tree/master/content/docs/game-references) or just doing a special "peds" folder on game-references and adding everything related to them in the future @4mmonium (that would break fivem-peds link page tho (if we moving it)

Reply:
Hello! I agree that this info should be somewhere within the docs, let me know abou your decision and any changes you want me to make!

I could also help setting up the page/section for this info.